src/lib/formats/dmk_dsk.cpp

```diff
@@ -83,6 +81,27 @@ int dmk_format::identify(util::random_read &io, uint32_t form_factor, const std:
 }
 
 
+uint32_t dmk_format::wide_fm(uint32_t val)
+{
+	uint32_t res = 0;
+	for (int i = 15; i >= 0; i--) {
+		if (val & (1 << i))
+			res |= (1 << (i*2 + 1));
```

Comment:
FYI, MAME also has a `BIT` utility: `if (BIT(val, i))`. Also alternatively you could get rid of the conditional here by doing something like `res |= BIT(val, i) << (i*2 + 1)`.

Reply:
Changed to use BIT and skip the "if" statement as described.